settings.gradle.kts

```
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
-plugins { id("org.hiero.gradle.build") version "0.3.1" }
+plugins { id("org.hiero.gradle.build") version "0.3.3" }
```

```suggestion
plugins { id("org.hiero.gradle.build") version "0.3.4" }
```

(but you can also do another update later)